src/index.html

```diff
@@ -5,8 +5,343 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="./index.html">
+        <img
+          src="./images/logo.png"
+          alt="MOYO_LOGO"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="https://www.apple.com/" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="https://www.samsung.com/ua/" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Laptops&Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="https://www.apple.com/" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="https://www.apple.com/" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="https://www.apple.com/" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="https://www.apple.com/" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="shop">
+      <div class="card" data-qa="card">
+        <div class="card__top">
+          <img
+            src="./images/imac.jpeg"
+            alt="img"
+            class="card__image"
+          >
+        </div>
+        <div class="card__bottom">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <div class="card__code">
```

Comment:
Why do you need this wrapper? Remove it and move CSS props that you need to card__subtitle

Reply:
Done. 

The idea was 
code for block position
subtitle for text styles